engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -557,6 +557,16 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
         }
     }
 
+    it should "include totalResultCount in workflow query response" in {
```

Comment:
Is there any way to be sure that this runs at exactly the right time for the answer to be `1`?

(ie to avoid the "unit tests might run in any order" thing)